h/models/organization.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+
+import sqlalchemy as sa
+import slugify
+
+from h.db import Base
+from h.db import mixins
+
+
+ORGANIZATION_NAME_MIN_CHARS = 1
+ORGANIZATION_NAME_MAX_CHARS = 25
+ORGANIZATION_LOGO_MAX_CHARS = 10000
```

Comment:
FWIW the "BioPub" logo used by the demo site at https://biopub.hypothes.is/ is 4963 bytes un-minified and according to https://jakearchibald.github.io/svgomg/ will reduce in size by ~40% when optimized. The other logos are ~2KB each unoptimized.

```sh
 ~/h/r/b/icons (autojump)> ls
logo.svg            search.svg          social-facebook.svg social-linkedin.svg social-twitter.svg
 ~/h/r/b/icons > cat logo.svg | wc -c
    4963
 ~/h/r/b/icons > cat social-facebook.svg | wc -c
    1289
 ~/h/r/b/icons > cat social-twitter.svg | wc -c
    2176
 ~/h/r/b/icons > cat social-linkedin.svg | wc -c
    1995
 ~/h/r/b/icons >
```